Vagrantfile

```diff
@@ -1,33 +1,70 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+
+ansible_vars = YAML.load_file('provisioning/vars.yml')
+
+app_name = ansible_vars["nodejs_app_name"]
+
+app_directory = ansible_vars["nodejs_app_install_dir"]
+
+# Check for the existence of 'VM_HOST_TCP_PORT' or 'VM_GUEST_TCP_PORT'
+# environment variables. Otherwise if 'nodejs_app_tcp_port' is defined
+# in vars.yml then use that port. Failing that use defaults provided
+# in this file.
+host_tcp_port = ENV["VM_HOST_TCP_PORT"] || ansible_vars["nodejs_app_tcp_port"] || 8080
```

Comment:
@amb26, @jobara the ``VM_*_TCP_PORT`` environment variables for overriding ports were provided so someone could launch the VM and select a different port without having to edit ``vars.yml`` or changing the default in this ``Vagrantfile``. However I think they can be removed now since port conflicts are prevented. If someone is using 8088 with another process on their machine it they can still start this VM and just use the new port that Vagrant logs in its output at startup.